### ISSUE TITLE
After having reinstalling AdGuard Pro there are now 2 AGP VPN profiles.

### ISSUE THREAD

**Issue by X8716e** [AdguardTeam/AdguardForiOS#532]
![538d6ef0-7d86-4ad2-8e85-2bb22bbbaecd](https://user-images.githubusercontent.com/18148748/33233894-a4cc3d96-d1eb-11e7-9b91-6dc3f6ee7b02.png)
![57f0faff-0151-48b6-8c34-8b0cbf55f980](https://user-images.githubusercontent.com/18148748/33233895-a4dbb9ec-d1eb-11e7-852a-842509e0a46a.png)
![3a573ded-1727-4e05-9166-135a7ffc4b47](https://user-images.githubusercontent.com/18148748/33233896-a4e8ae40-d1eb-11e7-85ee-ad6440c705e7.png)
![ebb9a1c2-9495-4767-8703-d14e4fb37813](https://user-images.githubusercontent.com/18148748/33233897-a4f5f1ea-d1eb-11e7-848d-f54e058d1b03.png)
![2fb1cd7d-8448-4081-b86f-37bf516b7366](https://user-images.githubusercontent.com/18148748/33233898-a501fc42-d1eb-11e7-9ac1-5f9ca5c8428c.png)
![c9f3be2c-ff7e-4cb5-899f-3b51a7a1371c](https://user-images.githubusercontent.com/18148748/33233899-a50f07de-d1eb-11e7-82c5-e690e218e2a9.png)

**Comment by X8716e** [comment]
See attached screenshots. I think this more of being interesting than of being an issue, though something that should be posted as an isuue nonetheless. I will return later to give any necessary additional details. Thank-you.
*note* The screenshots are a bit unorganized. What they utimately show is that I have one AG VPN that shows itself as on & connected to the custom dns.d0wn servers via port 5300. All pics w/ the VPN icon showing represent that VPN instance. The other (which is the clean install)

**Comment by X8716e** [comment]
Checking back in to see if any other info is needed. Let me know if there is.
_note--_ I did end up resolving this by performing a second clean reinstall. I tried downgrading then upgrading just in case that worked, but it did not. The new reinstall worked and all is good. I must say that I'm liking this version; it seems 'lighter' somehow & better able to handle custom DNS hosts.

**Comment by ameshkov** [comment]
@zebrum try reproducing plz

**Comment by X8716e** [comment]
I will as well, @ameshkov , though I suspect this particular anomaly is one that is quite rare, if not outright impossible to reproduce. Then again, that only makes me all the more eager to reproduce it.

**Comment by vozersky** [comment]
@X8716e did this happen after downgrading Adguard to the previous version?

**Comment by X8716e** [comment]
@voresky , Yes, though I only downgraded before doing a second clean install in the off-chance I'd not have to re-add my DNS blacklist.

**Comment by vozersky** [comment]
@IvanIin yep, looks like it is connected to downgrade. What can we do about it?

**Comment by ameshkov** [comment]
Guys, this occurs to me that the issue is simply too rare.

**Comment by X8716e** [comment]
Very. It did occur again; instead of commenting here, I immediately took screenshots & sent in log reports along with said screenshots.

**Comment by ameshkov** [comment]
@X8716e could you plz remind the customer ID we should look for?

**Comment by X8716e** [comment]
@ameshkov , my ID is **1600731**

**Comment by lancelot-moon** [comment]
The issue of 2 VPN profiles in log was also found in my topic.
(In VPN setting, it only shows one AdGuard Pro VPN.)
https://github.com/AdguardTeam/AdguardForiOS/issues/524#issuecomment-348980380

I have sent the bug report to support@adguard.com at 21:34 (GMT+8) on 4th Dec, 2017.
User ID: 1425202
Subject: [Adguard Pro for iOS] bug report
